tracer/dependabot/Datadog.Dependabot.Vendors.csproj

```diff
@@ -30,6 +30,21 @@
     <!-- https://www.nuget.org/packages/IndieSystem.Text.RegularExpressions/0.6 -->
     <PackageReference Include="IndieSystem.Text.RegularExpressions" Version="0.6" />
 
+    <!-- https://www.nuget.org/packages/System.Collections.Immutable/1.0.0 -->
+    <PackageReference Include="System.Collections.Immutable" Version="1.0.0" />
+
+    <!-- https://www.nuget.org/packages/System.Memory/1.0.0 -->
+    <PackageReference Include="System.Memory" Version="1.0.0" />
+
+    <!-- https://www.nuget.org/packages/System.Private.CoreLib/1.0.0 -->
+    <PackageReference Include="System.Private.CoreLib" Version="1.0.0" />
+
+    <!-- https://www.nuget.org/packages/System.Reflection.Metadata/1.0.0 -->
+    <PackageReference Include="System.Reflection.Metadata" Version="1.0.0" />
+
+    <!-- https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/1.0.0 -->
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="1.0.0" />
```

Comment:
We're _not_ vendoring these _directly_ though right? I don't think we should list them here if not, because otherwise we'll constantly get notifications that we should update them. And are we really based on 1.0.0? Which raises the question, when _should_ we update these? And if we're not based on 1.0.0, maybe we should update that vendoring source to follow the versions we _are_ vendoring, so that we can update them here, and get notifications about bugs?

Reply:
Fixed in ba3669bb94c1189959dc70349c641af1dfee264a